File: py/2894.py
def main():
    string = input()
    a, b = 0, 0

    for i in string:
        if i == '(':
            a += 1
        elif b < a:
            b += 1

    if a != b:
        print(f'Ainda temos {a - b} assunto(s) pendente(s)!')
    else:
        print('Partiu RU!')


main()

File: cpp/2894.cpp
#include <iostream>
#include <string>

using namespace std;

int main()
{
    string str;
    cin >> str;
    int a = 0, b = 0, i, l = str.length();

    for (i = 0; i < l; i++) {
        if (str[i] == '(') {
            a++;
        } else if (b < a) {
            b++;
        }
    }

    if (a != b) {
        cout << "Ainda temos " << a - b << " assunto(s) pendente(s)!\n";
    } else {
        cout << "Partiu RU!\n";
    }

    return 0;
}